keras/backend/numpy_backend.py

```diff
@@ -716,13 +722,16 @@ def _remove_blanks(inds, num_classes):
     return inds[inds < (num_classes - 1)]
 
 
+def expand_dims(x, axis=-1):
+    return np.expand_dims(x, axis=axis)
+
+
 square = np.square
 abs = np.abs
 exp = np.exp
 log = np.log
 round = np.round
 sign = np.sign
-expand_dims = np.expand_dims
```

Comment:
Let's make this modification in another PR to keep this structured, since this change seems unrelated to the goal of this PR.

Reply:
Sure, the only difference is that default args are not the same as in keras api.

First I used it in the `rnn` implementation - but then I figured it was better to stick to explicit numpy functions for the sake of the "numpy implementation as documentation".